sequencer/src/api/options.rs

```diff
@@ -184,6 +184,10 @@ pub struct Sql {
     /// Password for Postgres user.
     #[clap(long, env = "ESPRESSO_SEQUENCER_POSTGRES_PASSWORD")]
     pub password: Option<String>,
+
+    /// Reset database upon connecting.
+    #[clap(long, env = "ESPRESSO_SEQUENCER_RESET_STORE")]
```

Comment:
This is a bit annoying because it only supports `true` (not `1`, `True`, ...).

Reply:
@Ancient123 what's your preferred way to use this feature?

Reply:
true is fine, but supporting more options would be better.. Not worth holding up the PR though honestly.